config/toml_test.go

```diff
@@ -84,11 +77,6 @@ func checkConfig(configFile string) bool {
 		"genesis",
 	}
 	for _, e := range elems {
-		if !strings.Contains(configFile, e) {
-			valid = false
-		} else {
-			valid = true
-		}
+		assert.Contains(t, configFile, e)
```

Comment:
I think it's probably clearer from an error reporting perspective to have each of these elements be a subtest and then use require/fatal, rather than a bunch of asserts without demarcation. I'm also partial to not using assert, and tending toward using strings.Contains and a `Fatalf` assertion.

Reply:
Why do you prefer that exactly? I was going with the `assert`/`require` because I find the output clearer

Reply:
Not so much in this case because it was printing the entire configFile

Reply:
I just find the output easier to parse, as you'd see what the name of the file was that was incorrect and have the name of the subtest marked on the failure. Also testify's reference to the failure line isn't in a standard format so it can be hard to find. 

Not a blocking comment, anyway.